cloud/libvirt/actuators/machine/utils/domain.go

```diff
@@ -54,6 +55,24 @@ func newDomainDef() libvirtxml.Domain {
 		},
 		CPU: &libvirtxml.DomainCPU{},
 		Devices: &libvirtxml.DomainDeviceList{
+			Disks: []libvirtxml.DomainDisk{
```

Comment:
This is needed to run the `cloud-init` script living inside `/var/lib/libvirt/images/iso.iso`. Though, the path to the iso image needs to be parametrized.

Described based on `https://avdv.github.io/libvirt/formatdomain.html#elementsDisks`